src/app/components/task-button/task-button.component.ts

```diff
@@ -59,4 +61,17 @@ export class TaskButtonComponent implements OnInit {
       { display: 'inline-block' }
     );
   }
+  TaskButtonClick({dataJob, id} : {dataJob: any, id: number}): void {
+    switch (dataJob){
+    case this.constantsService.COMPLETE:
+      break;
+    case this.constantsService.EDIT:
+      break;
```

Comment:
Are we planning to do anything within these block? If not we can remove them?

Reply:
Suggestion: I you want to do something in the future, add a TODO: comment so that whoever reading the code understand the context. This will also help to track unfinished work.